docs/pkgs.nix

```diff
@@ -0,0 +1,56 @@
+{
+  system,
+  nixpkgs,
+}:
+let
+  # Building the docs evaluates each plugin's default package, some of which are unfree
+  # FIXME: is this still true? Most packages should only be evaluated for their meta.homepage URL now...
+  config = {
+    allowUnfree = true;
+  };
```

Comment:
I guess it _is_ still needed:

```
… while evaluating the default value of option `plugins.codeium-vim.settings.bin`

error: Package ‘codeium-1.10.0’ in /nix/store/XXX-source/pkgs/by-name/co/codeium/package.nix:51 has an unfree license (‘unfree’), refusing to evaluate.
```

We sometimes use the package-option value in the default for another option; in this case we're probably doing `lib.getExe cfg.package`.

Reply:
We can probably remove the "allow unfree" if we do a treewide adding `defaultText` for any options whose default evaluates a package-option's value.

But that's beyond the scope of this PR.

It would allow us to go back to using `pkgs.extend` instead of importing a fresh nixpkgs instance here, though.